stm32cube/stm32u5xx/drivers/src/stm32u5xx_ll_utils.c

```diff
@@ -66,9 +66,9 @@
 #define UTILS_SCALE2_LATENCY1_FREQ    (50000000U)      /*!< HCLK frequency to set FLASH latency 1 in power scale 2 */
 #define UTILS_SCALE2_LATENCY2_FREQ    (75000000U)      /*!< HCLK frequency to set FLASH latency 2 in power scale 2 */
 #define UTILS_SCALE2_LATENCY3_FREQ    (100000000U)     /*!< HCLK frequency to set FLASH latency 3 in power scale 2 */
-#define UTILS_SCALE3_LATENCY0_FREQ    (12.5000000)    /*!< HCLK frequency to set FLASH latency 0 in power scale 3 */
+#define UTILS_SCALE3_LATENCY0_FREQ    (125000000U)    /*!< HCLK frequency to set FLASH latency 0 in power scale 3 */
 #define UTILS_SCALE3_LATENCY1_FREQ    (25000000U)      /*!< HCLK frequency to set FLASH latency 1 in power scale 3 */
-#define UTILS_SCALE3_LATENCY2_FREQ    (37.5000000)    /*!< HCLK frequency to set FLASH latency 2 in power scale 3 */
+#define UTILS_SCALE3_LATENCY2_FREQ    (375000000U)    /*!< HCLK frequency to set FLASH latency 2 in power scale 3 */
```

Comment:
Could it be 37500000U  instead of  375000000U  ( 10 times higher)
```
#define UTILS_SCALE3_LATENCY1_FREQ    (25000000U)  
#define UTILS_SCALE3_LATENCY2_FREQ    (37500000U)  

```